src/ontology/mondo-edit.obo

```diff
@@ -302116,7 +302116,7 @@ synonym: "non-alcoholic fatty liver" EXACT [DOID:0080546, MONDO:0007026]
 synonym: "non-alcoholic fatty liver disease" EXACT [DOID:0080208, NCIT:C84444]
 synonym: "nonalcoholic fatty liver disease" EXACT [DOID:0080208, NCIT:C84444]
 xref: DOID:0080208 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
-xref: DOID:0080546 {source="MONDO:equivalentTo"}
+xref: DOID:0080546 {source="MONDO:equivalentObsolete"}
```

Comment:
This is not obsolete in DOID: https://www.ebi.ac.uk/ols4/ontologies/doid/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FDOID_0080208